restart_services.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+FILE=".env"
+inotifywait -m -e close_write $FILE |
+while read EVENT;
```

Comment:
```suggestion
inotifywait -m -e close_write $FILE | while read EVENT
```

Optional, minor nitpick as I didn't understand immediately you were feeding the while loop with the output from `inotifywait`.

Reply:
Thanks. Fixed it as it improves readability.